src/addons/SuggestionsWrapper.jsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { getClassName } from '../utils/helper';
+import { noSuggestions } from '../components/Input';
+
+const SuggestionWrapper = ({
+  theme,
+  themePreset,
+  children,
+  innerClassName,
+  innerClass
+}) => (
+  <div
+    className={`${noSuggestions(themePreset, theme)} ${getClassName(
+      innerClass,
+      innerClassName || ''
+    )}`}
+  >
+    <li>{children}</li>
```

Comment:
Let us not put `li` here. You can simply render inside the `div`. Currently, it's breaking the styles, you can reproduce using: 

```
renderNoSuggestion={() => <p>No Suggestion</p>}
```